server/chat-commands/admin.ts

```diff
@@ -16,6 +16,8 @@ import {Utils} from '../../lib/utils';
 
 import * as ProcessManager from '../../lib/process-manager';
 
+const PRIVATE_CODE_PATH = '../pokemon-showdown-private/';
```

Comment:
not sure about committing this constant (why is this not on `Config`?) especially given its actually incorrect (its called `main-private` on the box, not `pokemon-showdown-private`)...

Reply:
Sorry, I didn't know that. I'll move it to Config.